src/internal/connector/discovery/discovery.go

```diff
@@ -20,13 +19,17 @@ type getter interface {
 	GetByID(context.Context, string) (models.Userable, error)
 }
 
-type getInfoer interface {
+type GetInfoer interface {
 	GetInfo(context.Context, string) (*api.UserInfo, error)
 }
 
 type getWithInfoer interface {
 	getter
-	getInfoer
+	GetInfoer
+}
+
+type getAller interface {
```

Comment:
noob question: why is this getAller and not GetAller like line #22 above? Is this a naming pattern?

Reply:
There's no immediate need to export it.  In case you haven't looked into golang var exporting yet: https://medium.com/golangspec/exported-identifiers-in-go-518e93cc98af